Do not build the mkdata tool

The mkdata tool is needed on the host, during the build, so building
it for the target does not work when cross-compiling. Buildroot will
manually build mkdata before starting the ympd build.

Upstream is looking into a proper solution.

Signed-off-by: Eric Limpens <Limpens@Gmail.com>

Index: b/CMakeLists.txt
===================================================================
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -28,9 +28,6 @@
 	htdocs/index.html
 )
 
-add_executable(mkdata htdocs/mkdata.c)
-get_target_property(MKDATA_EXE mkdata LOCATION)
-
 add_custom_command(OUTPUT ${PROJECT_BINARY_DIR}/assets.c
     COMMAND ${MKDATA_EXE} ${RESOURCES} > ${PROJECT_BINARY_DIR}/assets.c
     WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
